usr/sbin/regionsrv-enabler-azure

```diff
@@ -97,20 +98,33 @@ utils.start_logging()
 service_name = 'guestregister'
 license_type = get_license_type()
 
-if ('BYOS' in license_type and has_license_changed(license_type) and
-    not utils.uses_rmt_as_scc_proxy()):
+if ('BYOS' in license_type and not utils.uses_rmt_as_scc_proxy()):
+    try:
+        if not has_license_changed(license_type):
```

Comment:
The `has_license_changed()` function,  https://github.com/SUSE-Enceladus/cloud-regionsrv-client/blob/master/usr/sbin/regionsrv-enabler-azure#L32, does not throw, there is no reason to have the call wrapped in a `try - except` block.

Reply:
We could add a `#Still BYOS nothing to do` comment here

Reply:
> The `has_license_changed()` function, https://github.com/SUSE-Enceladus/cloud-regionsrv-client/blob/master/usr/sbin/regionsrv-enabler-azure#L32, does not throw, there is no reason to have the call wrapped in a `try - except` block.

The re raise of the exception is added in this [PR](https://github.com/SUSE-Enceladus/cloud-regionsrv-client/blob/a8d053d0ba259b57473922d1339f231845049153/usr/sbin/regionsrv-enabler-azure#L40) ,
as the method should return either `True` or `False` but on that case was returning `None` when it actually 
can not know if it did change

 We could leave the method returning `None`, the approach would be very similar

Reply:
No exception re raised, using `None` value as different to `False`